jaxrs/src/main/java/org/killbill/billing/jaxrs/json/CatalogJson.java

```diff
@@ -87,6 +87,7 @@ public CatalogJson(final StaticCatalog catalog) throws CatalogApiException {
         currencies = Arrays.asList(catalog.getSupportedCurrencies());
         priceLists = new ArrayList<PriceListJson>();
 
+        final Collection<Product> XMLproducts = catalog.getProducts();
```

Comment:
The variable is not named as per coding conventions, typically variable name should start with a lowercase letter and follow camelcase. Also, it has nothing to do with XML. So consider renaming to `catalogProducts` for clarity.